jax/_src/lax/special.py

```diff
@@ -45,6 +45,10 @@ def lgamma(x: ArrayLike) -> Array:
   r"""Elementwise log gamma: :math:`\mathrm{log}(\Gamma(x))`."""
   return lgamma_p.bind(x)
 
+def gamma(x: ArrayLike) -> Array:
+  r"""Elementwise gamma: :math:`\Gamma(x)`."""
+  return exp(lgamma_p.bind(x))
```

Comment:
I would probably avoid adding any new function here, unless it's a nontrivial implementation.

Reply:
I am sorry, I did not see this before adding docs... I think I need to reopen the PR and fix it